### ISSUE TITLE
website consuming much cpu

### ISSUE THREAD

**Issue by tkkcc** [specta-rs/rspc#302]
i am on chromium 127.0.6533.99 on archlinux, [rspc.dev](https://rspc.dev/) quickstart page consumes much cpu. even when tab is unfocused. close tab will solve. disable all chromium plugins not help. i have no similar issue visiting other sites in recent years.

https://github.com/user-attachments/assets/f23ec090-5826-4234-81e0-285f8a9c26ee

**Comment by oscartbeaumont** [comment]
I think it's being caused by [this line](https://github.com/oscartbeaumont/rspc/blob/50429522a94206b1f7fc4eec404d8b6d91a92873/docs/components/Switchers.tsx#L90) introduced in #283.

**Comment by oscartbeaumont** [comment]
Yeah. In practice this isn't fixed but we are dropping the existing documentation side (`rspc.dev`) and merging it into `specta.dev`.
